src/Plugin/TestsiteBuilder/ConfigTemplateType/ViewConfigCacheTag.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\testsite_builder\Plugin\TestsiteBuilder\ConfigTemplateType;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\testsite_builder\ConfigTemplateMerge;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * View config cache tag config template type plugin.
+ *
+ * @TestsiteBuilderConfigTemplateType(
+ *   id = "view_config_cache_tag",
+ *   label = @Translation("View config cache tag"),
+ *   description = @Translation("View config cache tag config template type plugin.")
+ * )
+ */
+class ViewConfigCacheTag extends Generic {
+
+  /**
+   * The entity type manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager) {
```

Comment:
you could scrap the __constructor here and in ViewDependencyConfig.php and change the ::create() to 
'
   public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
    $plugin = parent::static($configuration, $plugin_id, $plugin_definition);
    $plugin->entityTypeManager = $container->get('entity_type.manager');
    return $plugin;
  } 
'